pyproject.toml

```diff
@@ -1,3 +1,40 @@
+[project]
```

Comment:
I pretty much copied this section from [the example in PEP 621](https://peps.python.org/pep-0621/#example).